app/assets/javascripts/cart.js.coffee

```diff
@@ -45,6 +42,8 @@ $(document).ready ->
       $('#modal_place').modal('show')
       $('#modal_place .yes-button').data('srid', srid)
       $('#modal_place .yes-button').data('service-id', id)
+      $('#modal_place .yes-button').on 'click', (e) ->
```

Comment:
Is this a local handler? I noticed that the problem with the original code was that as you would remove/add more services, the handlers would add up and multiple clicks would be registered when you click the 'Yes' button. @jwiel86 Can you confirm?

If this does work, we should remove the two above lines setting data attributes on the yes button because they won't be needed anymore